src/frontend/src/drakulaAi3nd.js

```diff
@@ -0,0 +1,121 @@
+import { getGlobals, updateGlobals } from './testMainGameLoop.js';
```

Comment:
Renaming the file to `ai.js` would be nice

Reply:
let's forget drakulaAi3nd.js